src/components/organisms/navbar.js

```diff
@@ -1,19 +1,39 @@
-import React from "react"
+import React, { useState } from "react"
 import PropTypes from "prop-types"
 import Navigation from "../molecules/navigation"
+import { Header } from "../../styles/header-styles"
+import blueRingImage from "../../images/logos/ring-blue.png"
 
-const NavBar = () => (
-  <header>
-    <Navigation />
-  </header>
-)
+const NavBar = ({ isHome }) => {
+  const [hamburgerState, setHamburgerState] = useState(false)
+  console.log({ hamburgerState })
```

Comment:
Thoughts on removing this `console.log`?

Reply:
I have removed the console log in the review file now. :)